### ISSUE TITLE
[Bug] Autodocs produce invalid code example

### ISSUE THREAD

**Issue by sacrosanctic** [storybookjs/addon-svelte-csf#166]
### Describe the bug

`show code` section produces code examples that look like `<Proxy<RenderContext> />` when using the following:

```html
<Template let:args>
  <MyComponent {...args} />
</Template>
```

### Steps to reproduce the behavior

[Repro](https://github.com/sacrosanctic/repro-sb-autodocs)

1. `npm i`
2. `npx storybook@latest init`
3. `npm i --save-dev @storybook/addon-svelte-csf`
4. add stories
5. `npm run storybook`
6. view [`NotWorking`](http://localhost:6006/?path=/docs/routes-notworking--docs) and [`Working`](http://localhost:6007/?path=/docs/routes-working--docs) and click `show code`

### Expected behavior

Should say:
- `<MyComponent />`

### Screenshots and/or logs

![image](https://github.com/storybookjs/addon-svelte-csf/assets/2974920/5953ee27-1e54-4ecc-b136-38571b0824aa)

**Comment by j3rem1e** [comment]
When you use "args" stories, the code is generated by Storybook (and not this addon)
When you don't use "args" stories, the code is extracted from the svelte template.

the "Proxy<..>" is used by svelte-hmr when vite is used, and the component can't be parsed by Storybook/svelte-docgen parser. However, I thought this issue had been fixed in storybook in the v7.6 branch.

**Comment by j3rem1e** [comment]
Well, sorry. but nothing I can do on my side, so i'll close this issue.

it should only happen in dev mode, when the vite hmr is used, not in a real prod build. The vite proxy doesn't have a way to get the added hidden __docgen field. You can maybe create an issue in the storybook repo, but I think there are already severals.
